protocol/implementations/js/src/types.ts

```diff
@@ -0,0 +1,32 @@
+export namespace TbDex {
+  export type Offering = {
+    placeholder: string;
+  }
+  
+  export type RFQ = {
+    test: string;
+  }
+  
+  export type Quote = {
+    placeholder: string;
+  }
+  
+  export type MessageTypes = {
+    rfq: RFQ,
+    quote: Quote
+  }
+  
+  export type MessageType<M extends keyof MessageTypes> = MessageTypes[M];
+  
+  export type Message<T extends keyof MessageTypes> = {
+    id: string;
+    contextId: string;
+    from: string;
+    to: string;
+    type: T;
+    body: MessageTypes[T];
+    dateCreated: string;
```

Comment:
there's an `expiresTime` in the current readme - are we not using that field now? (tbh i was never 100% sure what that field would be for. the only time we use expiry is when we give out a Quote, and that would live in the `body` field anyway).

Reply:
💯 agreed. when i saw it i couldn't remember why i added it at the top-level because like you mentioned, expirations are really only needed for quotes

Reply:
ok. i'll just write a smol issue to get rid of that field at the top tbdex message field level in the readme